lessons/intro/distribution/index.md

```diff
@@ -254,27 +263,38 @@ setup(
 )
 ```
 
+> [note]
+> A jaký je tedy vlastně rozdíl mezi `py_modules` a `packages`?
+> [Ten první není hierarchicky nořen][so].
+
+[so]: https://stackoverflow.com/questions/7948494/whats-the-difference-between-a-python-module-and-a-python-package
+
```

Comment:
Ale je tohle důležité i v kontextu `setup.py`?
A nebylo by nakonec lepší tam dát už od začátku `packages`, ať to lidi nemate?

Reply:
To je taky řešení.

Reply:
(jdu to udělat)

Reply:
Není to tak jednoduché. packages s jedním souborem nefunguje. Revertoval jsem to. Ladit můžeme později.